src/TTImp/ProcessBuiltin.idr

```diff
@@ -80,10 +80,7 @@ getNEIndex : (arity : Nat) -> Term vars -> Maybe (Fin arity)
 getNEIndex ar (Bind _ x b tm) = case b of
     Let _ _ val _ => getNEIndex ar $ subst {x} val tm
     Pi _ mul _ arg => if isErased mul
-        then getNEIndex ar tm >>=
-            \k => case strengthen (FS k) of
-                Left _ => Nothing
-                Right k' => Just k'
+        then getNEIndex ar tm >>= \k => Libs.strengthen (FS k)
```

Comment:
```suggestion
        then getNEIndex ar tm >>= Libs.strengthen . FS
```

Not really a requirement but checkout out your branch and just realized it is easy to turn it dot-free.